SQLScripts/US004 - Populate Bands Table.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO TABLE Bands(band_name) VALUES('Trainee');
+INSERT INTO TABLE Bands(band_name) VALUES('Apprentice');
+INSERT INTO TABLE Bands(band_name) VALUES('Associate');
+INSERT INTO TABLE Bands(band_name) VALUES('Senior Associate');
+INSERT INTO TABLE Bands(band_name) VALUES('Consultant');
+INSERT INTO TABLE Bands(band_name) VALUES('Manager');
+INSERT INTO TABLE Bands(band_name) VALUES('Principal');
+INSERT INTO TABLE Bands(band_name) VALUES('Leadership');
```

Comment:
Would something like this work instead? 

```suggestion
INSERT INTO TABLE Bands(band_name) 
VALUES 
     ('Trainee'),
     ('Apprentice'),
     ('Senior Associate'),
     ...
```